pkg/api/configsync/v1alpha1/resource_override.go

```diff
@@ -54,6 +54,14 @@ type OverrideSpec struct {
 	// +optional
 	ReconcileTimeout *metav1.Duration `json:"reconcileTimeout,omitempty"`
 
+	// apiServerTimeout allows one to override the client-side timeout for requests to the API server.
```

Comment:
You need to run `make generate` here to update `pkg/api/configsync/v1alpha1/zz_generated.deepcopy.go`

Reply:
Thanks!